lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -775,7 +775,7 @@ class DefaultAssetPickerBuilderDelegate
           child: Text(
             provider.isSelectedNotEmpty && !isSingleAssetMode
                 ? '${Constants.textDelegate.confirm}'
-                    '(${provider.selectedAssets.length}/${provider.maxAssets})'
+                    ' (${provider.selectedAssets.length}/${provider.maxAssets})'
```

Comment:
This space is required to make sure the parenthesis are not too close to the text